pytorch/inference/docker/build_artifacts/mms-entrypoint.py

```diff
@@ -15,14 +15,17 @@
 import shlex
 import subprocess
 import sys
+import os
+import multiprocessing
 
+os.environ["SAGEMAKER_MODEL_SERVER_WORKERS"] = str(multiprocessing.cpu_count())
```

Comment:
This should happen only if `SAGEMAKER_MODEL_SERVER_WORKERS` is not set; otherwise there is a risk of loading multiple copies of the same model and going beyond RAM/GPU RAM limits. Also probably a good idea to have upper cap (i.e. if there is a large instance with lots of cores, it's not necessarily optimal to have the same number of workers).